unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/AssetManager/GLTFast/AssetPromise_GLTFast_Loader.cs

```diff
@@ -143,6 +143,50 @@ private async UniTaskVoid ImportGltfAsync(Action onSuccess, Action<Exception> on
             contentProvider.TryGetContentsUrl(assetDirectoryPath + fileName, out fileHash);
     }
 
+    public class GltfastEditorLogger : ICodeLogger
+    {
+        public LogCode LastErrorCode { get; private set; }
+
+        public void Error(LogCode code, params string[] messages)
+        {
+            LastErrorCode = code;
+            Debug.LogError(LogMessages.GetFullMessage(code, messages));
+        }
+
+        public void Warning(LogCode code, params string[] messages)
```

Comment:
you can try to use `[Conditional("UNITY_EDITOR")]` attribute in these cases. It will be almost the same. Some info about it:

> The [Conditional] attribute in C# is an feature that enables developers to control whether the call to a particular method should be included or excluded at compile time, based on the presence or absence of a specified preprocessor symbol. 
This is extremely useful in cases where you have code that is only relevant in debug mode or during testing, such as writing debug logs to the console.

The main points to remember about the Conditional attribute are:
- It only affects whether calls to the method are included, not whether the method itself is included in the compiled code.
- The method that has the Conditional attribute must return void.
- You can apply the Conditional attribute to a method or attribute class, but not to an instance constructor, destructor, operator, or get or set property accessor.
- Any arguments passed to a method that is conditionally excluded are still fully compiled and executed; the call to the method is the only thing that is excluded.

Reply:
I was not aware of that Conditional Attribute, It also leaves the code more readable, nice suggestion! 👍

Reply:
Amazing stuff! Didn't know either!